ts/typegen/cmd/main.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019 Google LLC
```

Comment:
super nit: Is this auto-generated by some script? If so, may need to update to 2022